rest-tests/src/test/java/io/syndesis/qe/rest/tests/integrations/TwSfSteps.java

```diff
@@ -0,0 +1,124 @@
+package io.syndesis.qe.rest.tests.integrations;
+
+import static org.junit.Assert.assertThat;
+
+import static org.hamcrest.core.Is.is;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.security.GeneralSecurityException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+
+import cucumber.api.java.en.Given;
+import io.syndesis.model.connection.Connection;
+import io.syndesis.model.connection.Connector;
+import io.syndesis.model.filter.FilterPredicate;
+import io.syndesis.model.filter.RuleFilterStep;
+import io.syndesis.model.integration.Integration;
+import io.syndesis.model.integration.SimpleStep;
+import io.syndesis.model.integration.Step;
+import io.syndesis.qe.accounts.AccountsDirectory;
+import io.syndesis.qe.endpoints.ConnectionsEndpoint;
+import io.syndesis.qe.endpoints.ConnectorsEndpoint;
+import io.syndesis.qe.endpoints.IntegrationsEndpoint;
+import io.syndesis.qe.utils.FilterRulesBuilder;
+import io.syndesis.qe.utils.RestConstants;
+import io.syndesis.qe.utils.TestUtils;
+import lombok.extern.slf4j.Slf4j;
+
+/**
+ * Steps for Twitter mention to Salesforce upsert contact integration.
+ *
+ * Oct 7, 2017 Red Hat
+ *
+ * @author tplevko@redhat.com
+ */
+@Slf4j
+public class TwSfSteps {
+
+	public static final String INTEGRATION_NAME = "Twitter to salesforce contact rest test";
+	public static final String SYNDESIS_TALKY_ACCOUNT = "twitter_talky";
+
+	private ConnectionsEndpoint connectionsEndpoint;
+	private ConnectorsEndpoint connectorsEndpoint;
+	private final List<Step> steps = new ArrayList<>();
+	private AccountsDirectory accountsDirectory;
+
+	public TwSfSteps() throws GeneralSecurityException {
+		accountsDirectory = new AccountsDirectory();
+		connectorsEndpoint = new ConnectorsEndpoint(RestConstants.getInstance().getSyndesisURL());
+		connectionsEndpoint = new ConnectionsEndpoint(RestConstants.getInstance().getSyndesisURL());
+	}
+
+	@Given("^user creates tw mention step with \"([^\"]*)\" action")
```

Comment:
Maybe we could have one generic method  like`user creates step from connector \"([^\"]*)\" with \"([^\"]*)\" action`